doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +57,237 @@ length, :math:`L_0`,
+
+    h(T_h) = 5.0 \left(1 + \left((2\times 10^{-6}(1000 − 20)\right)\right) = 5.01 cm
+
+.. math::
+   :name: hot_volume
+   
+    V(T_h) = \pi (0.251)^2 5.01 = 0.988 cm^3
+
+.. math::
+   :name: hot_density
+
+    \rho(T_h) = \frac{1.0}{\left(1 + 2\times 10^{-6}(1000 − 20)\right)} = 0.994 \frac{g}{cc}
+
+.. math::
+   :name: hot_mass
+
+    m(T_h) = 0.994 \times 0.988 = 0.982 g
+
+Radial thermal expansion occurs for each component in a given block. Mechanical contact between
+components is not accounted for, meaning that the radial expansion of one Component is independent
+from the radial expansion of the others. Solid components may be linked to gas/fluid components
+(i.e., sodium bond, helium) and the gas/fluid area is allowed to expand and contract with changes in
+Component temperatures. It is worth noting that void components are allowed to have negative areas
+in cases where the expansion of two solid components overlap each other.
+
+When two or more components exist within the Block, the overall height change of the Block is driven
+by an axial expansion "target Component" (e.g., fuel). Alternatively, the axial thermal expansion of
+components within a Block influences the positions of components in axially neighboring blocks for a given
+Assembly. So, while axial thermal expansion evaluations of each Block are treated independently, the
+axial mesh points are updated to account for the physical material displacements across the entire
+assembly length.
+
+The following two tables provide illustrations of the axial thermal expansion process for an example
+core assembly. In this example there are four main block types defined: Shield, Fuel, Plenum, and
+Dummy.
+
+.. figure:: /.static/axial_expansion_components.png
+   :scale: 60%
+
+    Illustration of Components for Axial Thermal Expansion Process
+
+.. figure:: /.static/axial_expansion_process.png
+
+    Simplified Illustration of Axial Thermal Expansion Process for a Core Assembly
+
+The target components for each Block type are provided in the following table:
+
+.. list-table:: Example Assignment of Target Components within Blocks
+   :widths: 50 50
+   :header-rows: 1
+
+   * - Block
+     - Target Component 
+   * - Shield
+     - Shield
+   * - Fuel
+     - Fuel
+   * - Plenum
+     - Clad
+   * - Dummy
+     - N/A
+
+The axial thermal expansion algorithm is applied in four steps:
+
+# Expand the axial dimensions of target component and non-target components within each
+  block independently. 
+# Align blocks axially such that common components have consistent alignments (e.g.,
+  overlapping radial dimensions).
+# Assign the block lower and upper elevations to account for the thermal expansion of blocks
+  below each block.
+
+   #  Create new mesh lines that track the target component.
+
+# Adjust the "Dummy" block located at the top of the assembly to maintain a consistent
+  core-wide assembly height before and after axial thermal expansion is applied.
```

Comment:
```suggestion
#. Expand the axial dimensions of target component and non-target components within each
  block independently. 
#. Align blocks axially such that common components have consistent alignments (e.g.,
  overlapping radial dimensions).
#. Assign the block lower and upper elevations to account for the thermal expansion of blocks
  below each block.

   #.  Create new mesh lines that track the target component.

#. Adjust the "Dummy" block located at the top of the assembly to maintain a consistent
  core-wide assembly height before and after axial thermal expansion is applied.
```
This should fix this
![image](https://github.com/user-attachments/assets/25937847-1ff1-4355-b146-4e1bfcd31dd8)